src/app/views/form/form.component.ts

```diff
@@ -1,24 +1,40 @@
 import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
 
-interface Mentor {
-  value: string;
-  viewValue: string;
-}
+import { Student } from 'src/app/shared/student';
+import { StudentService } from '../../services/student.service';
 
 @Component({
   selector: 'app-form',
   templateUrl: './form.component.html',
   styleUrls: ['./form.component.scss']
 })
 export class FormComponent implements OnInit {
-  mentor: Mentor[] = [
-    {value: 'Petras Petraitis', viewValue: 'Petras Petraitis'},
-    {value: 'Vardenis Pavardenis', viewValue: 'Vardenis Pavardenis'},
-    {value: 'Auksinis Kardas', viewValue: 'Auksinis Kardas'}
-  ];
-
-  constructor() {}
+  student: Student[];
+  selectedValue: string;
+  profileForm: FormGroup;
+  constructor(private studentService: StudentService,private fb: FormBuilder) {}
 
   ngOnInit(): void {
+    this.profileForm = this.fb.group({
+      studentId: [''],
+      participation: [''],
+      techSkills: [''],
+      learningPace: [''],
+      extraMile: [''],
+      comment: ['']
+      }),
+    this.getStudent();
   }
+
+  onSubmit(){
+  }
+
+  getStudent() {
```

Comment:
Is this page a form for a single student evaluation or to display all students? It seems that these should be separate. And I suppose you want to start fetching students before the page is loaded, in such case you could use a guard to fetch students.

Reply:
This is not done yet, for now we use getStudent() method to get students from dropdown. For showing all students in home page I think we will use different component and separate it.

Reply:
And yes this is page form for single student evaluation